README.md

```diff
@@ -85,7 +86,23 @@ $ make docker-build
 
 ## Deployment
 
-### Kubernetes
+### Harbor 2.0 on Kubernetes
+
+In Harbor >= 2.0 Trivy can be configured as the default vulnerability scanner, therefore you can install it with the
+official Harbor Helm chart, where `HARBOR_CHART_VERSION` >= 1.4:
```

Comment:
Should the Helm chart version be >= 2.0?

Reply:
There're two versions: app version = 2.0 and Helm chart version = 1.4

Reply:
```
$ helm search repo harbor/harbor                                                                                            [13:57:20]
NAME         	CHART VERSION	APP VERSION	DESCRIPTION                                       
harbor/harbor	1.4.0        	2.0.0      	An open source trusted cloud native registry th...
```